omnibus/Gemfile

```diff
@@ -9,7 +9,6 @@ gem "pedump"
 # This development group is installed by default when you run `bundle install`,
 # but if you are using Omnibus in a CI-based infrastructure, you do not need
 # the Test Kitchen-based build lab. You can skip these unnecessary dependencies
-# by running `bundle install --without development` to speed up build times.
```

Comment:
would it help to remove 'You can skip these unnecessary dependencies' as well in the line above? or replace the comment with something like 'You can skip these unnecessary dependencies by running `bundle config set without dep_selector` to speed up build times.' ?